### DISCUSSION TITLE
Improved schedule reliability using managed solution or fallback capability?

### DISCUSSION THREAD

**Discussion by SHARANTANGEDA** [cuebook/CueObserve#169] [Ideas]
Hi Team,
Is there any plan to support more reliable task scheduling framework compared to celery?. I know once upon a time celery was pretty stable, but doesn't seem like it now(https://github.com/cuebook/CueObserve/issues/147). So wanted to know is there any plan to support another task scheduler which is more reliable?

On a separate note, in case this celery issue is going to be here for a while, do we have an option to allow fallback schedule or auto retry when celery fails with above issue?

**Comment by sachinkbansal** [comment]
@SHARANTANGEDA which task scheduler would you prefer?

**Reply by SHARANTANGEDA** [reply]
@sachinkbansal  Well, I'm not sure if these are the solution that you would want to go with, but to put the managed option out there
1. We have Databricks, a job submitter can be created with Databricks Rest APIs, which will run the schedule and post the results to database
2. Azure Logic Apps is another solution in azure platform
3. Kubernetes Jobs

But Databricks is the best solution, it is available in all major cloud platforms, and reliability wise is very good 
Jobs API => https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/jobs#--create

**Reply by SHARANTANGEDA** [reply]
Moreover, if databricks is something that you would want to go with, Spark enables to train prophet with very very large datasets (order of 100s of GB), even capability of Delta Lakes can be added easily.
